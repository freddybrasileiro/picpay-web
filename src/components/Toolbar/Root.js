import styled from "styled-components";

export default styled.section`
  height: 40px;
  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.5);
  background-color: #7de6c6;
  display: flex;
  align-items: center;
`;
