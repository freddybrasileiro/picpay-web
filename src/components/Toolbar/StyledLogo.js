import styled from "styled-components";

export default styled.section`
  margin-left: 8px;
`;
