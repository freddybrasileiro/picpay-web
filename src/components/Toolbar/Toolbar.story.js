import React from "react";
import { storiesOf } from "@storybook/react";
import { ThemeProvider } from "styled-components";
import { IntlProvider } from "react-intl";

import styledComponentsTheme from "styledComponentsTheme";
import { DEFAULT_LOCALE, translationMessages } from "i18n";

import Toolbar from "components/Toolbar";

storiesOf("Components/Toolbar", module)
  .addDecorator(story => (
    <IntlProvider
      locale={DEFAULT_LOCALE}
      messages={translationMessages[DEFAULT_LOCALE]}
    >
      <ThemeProvider theme={styledComponentsTheme}>
        <div style={{ maxWidth: 500, margin: "10px 10px" }}>{story()}</div>
      </ThemeProvider>
    </IntlProvider>
  ))
  .add("Default", () => <Toolbar />);
