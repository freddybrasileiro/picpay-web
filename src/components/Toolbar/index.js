import React from "react";
import Grid from "@material-ui/core/Grid";

import Root from "components/Toolbar/Root";
import StyledLogo from "components/Toolbar/StyledLogo";
import PicPay from "components/icons/PicPay";

const Toolbar = () => (
  <Root>
    <Grid
      container
      justify="center"
      alignItems="center"
      direction="row"
      spacing={8}
    >
      <Grid item xs={12} md={6}>
        <StyledLogo>
          <PicPay width="102" height="16" />
        </StyledLogo>
      </Grid>
    </Grid>
  </Root>
);

export default Toolbar;
