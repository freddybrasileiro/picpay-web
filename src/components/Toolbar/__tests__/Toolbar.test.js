import React from "react";
import { createShallow } from "@material-ui/core/test-utils";

import Toolbar from "components/Toolbar";

describe("Toolbar", () => {
  const shallow = createShallow({ dive: true });

  it("Should render the Toolbar", () => {
    expect(shallow(<Toolbar />)).toMatchSnapshot();
  });
});
