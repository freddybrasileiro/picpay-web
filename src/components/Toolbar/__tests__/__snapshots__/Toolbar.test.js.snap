// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar Should render the Toolbar 1`] = `
<section
  className="sc-bdVaJa leDNtK"
>
  <WithStyles(Grid)
    alignItems="center"
    container={true}
    direction="row"
    justify="center"
    spacing={8}
  >
    <WithStyles(Grid)
      item={true}
      md={6}
      xs={12}
    >
      <styled.section>
        <Component
          height="16"
          width="102"
        />
      </styled.section>
    </WithStyles(Grid)>
  </WithStyles(Grid)>
</section>
`;
